django/thunderstore/account/templates/profile.html

```diff
@@ -0,0 +1,62 @@
+{% extends 'base.html' %}
+{% load thumbnail %}
+{% load arrow %}
+{% load social %}
+
+{% block title %}Test{% endblock %}
+
+{% block content %}
+<div class="card bg-light mt-2 mb-2">
+    <div class="card-header">
+        Uploader profile for <b>{{ user.username }}</b>
+    </div>
+    <div class="card-body">
+        <div class="d-flex flex-row flex-wrap">
+            <div class="col-12 col-md-12 col-lg-8">
+                <img src="https://www.gravatar.com/avatar/{{ email_hash }}?s=200"
+                    class="profile-avatar" />
+                <h3><b>{{ user.username }}</b></h3>
+                <h4>
+                    <i class="fas fa-box text-dark mr-1"></i>&nbsp;{{ package_count }}&nbsp;&nbsp;
+                    <i class="fas fa-download text-dark mr-1"></i>&nbsp;{{ total_downloads}}&nbsp;&nbsp;
+                    <i class="fas fa-thumbs-up text-dark mr-1"></i>&nbsp;{{ total_likes }}
+                </h4>
+                <p>
+                    Join date: {{ user.date_joined|date:"DATE_FORMAT" }}
+                    <h5>Connections:</h5>
+                    {% for provider in social_providers %}
+                    <span class="text-capitalize">{{ provider.provider }}</span>: {% social_username user provider.provider %}<br>
+                    {% endfor %}
```

Comment:
As far as I know, these aren't already public and there should probably be a discussion about how we would want to introduce this. If there are users who expected their linked accounts to be private, we should default to having them be private and have a toggle for their visibility.

Having a verified list of identities is nice (compared to e.g. a bio which may be inaccurate), although providing just a username which could be stale might not be the best solution. Both GitHub and Discord let you change your username. On GitHub, your user ID is more hidden than Discord so including the ID probably won't help this problem.

Related: #334

Reply:
Having a toggle for them in the settings is probably good, yeah. I suppose it's possible to fetch their latest username if we have oauth perms too? Or just tell people if you update your username disconnect and reconnect the service.